codestyle/druid-forbidden-apis.txt

```diff
@@ -35,6 +35,11 @@ java.lang.Math#random() @ Use ThreadLocalRandom.current()
 java.util.regex.Pattern#matches(java.lang.String,java.lang.CharSequence) @ Use String.startsWith(), endsWith(), contains(), or compile and cache a Pattern explicitly
 org.apache.commons.io.FileUtils#getTempDirectory() @ Use org.junit.rules.TemporaryFolder for tests instead
 
+@defaultMessage Uses default locale
```

Comment:
"Use Locale.ENGLISH" would be more constructive

Reply:
I've updated my PR.

Reply:
@leventov is there anything left to improve my PR?